integrations/release/manifest.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Release",
+  "version": "1.0.0",
+  "overview": "Manages your LaunchDarkly environments when deploying your Release Environments.",
```

Minor: to be consistent with the language on other [integration cards](https://app.launchdarkly.com/default/integrations) let's change this to:

```suggestion
  "overview": "Manage LaunchDarkly environments when deploying Release Environments.",
```